Policies/github-additional-product-terms.md

```diff
@@ -48,9 +48,9 @@ Compute usage for included and paid quantities is calculated in minutes based on
 
 Actions and any elements of the Action service may not be used in violation of the Agreement, the [Acceptable Use Policy](/github/site-policy/github-acceptable-use-policies), or the GitHub Actions [service limitations](/github/automating-your-workflow-with-github-actions/about-github-actions#usage-limits). Additionally, Actions should not be used for:
 - cryptomining;
-- serverless computing;
 - using our servers to disrupt, or to gain or to attempt to gain unauthorized access to, any service, device, data, account or network (other than those authorized by the [GitHub Bug Bounty program](https://bounty.github.com))
-- the provision of a stand-alone or integrated application or service offering Actions or any elements of Actions for commercial purposes; or,
+- the provision of a stand-alone or integrated application or service offering Actions or any elements of Actions for commercial purposes; 
+- any activity that places an undue burden on our servers disproportionate to the benefits provided to users (for example, don't use Actions as a content delivery network or as part of a serverless application); or
```

Comment:
Is testing an action considered `undue burden on our servers disproportionate to the benefits provided to users`? Testing an action brings completely no direct benefit to the users of that action.

Is building a packages also considered `undue burden on our servers disproportionate to the benefits provided to users`? A package clearly can be built on the machine of its developer.

I have contacted the support about the definition of `serverless computing`, in the sense if automatical moderation of issues or complex multi-repo pipelines can be considered prohibited `serverless computing` and got the clarification that as long as I use the Actions for the purpose of automation workflows on GitHub, the activity is allowed.

Reply:
@KOLANICH - Thank you this feedback! We're thinking about how to make this clearer and will update here.

Reply:
>We're thinking about how to make this clearer and will update here.

I just pushed https://github.com/github/site-policy/pull/279/files#diff-8f11672a475e377724d388f8c90563cb